tyr/servers/server.py

```diff
@@ -806,8 +812,12 @@ def connection(self):
 
         while True:
             try:
-                connection.connect(self.instance.public_dns_name,
-                                    username = 'ec2-user')
+                keys = ['~/.ssh/stage', '~/.ssh/prod']
```

Comment:
Does this mean we need our stage and prod ssh keys named as such?  Will it work if i'm using ssh-agent to add manage multiple ssh keys?